include/benchmark/benchmark_api.h

```diff
@@ -616,9 +628,25 @@ class Benchmark {
   Benchmark(Benchmark const&);
   void SetName(const char* name);
 
+  int ArgsCnt() const;
+
+  static void AddRange(std::vector<int>* dst, int lo, int hi, int mult);
+
 private:
   friend class BenchmarkFamilies;
-  BenchmarkImp* imp_;
```

Comment:
you popped my pimpl!

i'm actually fine with this. originally this was here to avoid having too much detail in the public header.